app/assets/javascripts/application.js

```diff
@@ -36,5 +36,6 @@
 //= require toastr.min.js
 //= require ekko_lightbox.min.js
 //= require chartkick
+//= require Chart.bundle
```

Comment:
Where is Chart.bundle coming from? What does this do for us that adjusts dashboard logic?

Totally cool to includes it ; but, we should be sure not to mix adding new features with refactors/fixes